### ISSUE TITLE
Would be nice to add Chrome like behavior with bookmarks

### ISSUE THREAD

**Issue by NyanTyrrell** [brave/browser-laptop#2740]
Hello!
I'm sorry, if I can't write here about it. First time I try to write an issue. 
As I already said, would be nice to add Chrome like behavior with bookmarks.
Chrome has a really good option which hides bookmarks toolbar and shows it only when you want to add a new tab. I've never seen this possibility in another browsers though that's very comfortable.
Sorry for my English.

**Comment by bsclifton** [comment]
@NyanTyrrell can you confirm I've captured this properly? I want to make sure the feature request is clear, in case anyone wanted to grab this

- Hide bookmarks toolbar by default
- You open new tab
- New tab has the bookmarks toolbar shown

**Comment by bsclifton** [comment]
@NyanTyrrell could you verify the above steps I captured sound correct? Thanks :smile:

**Comment by NyanTyrrell** [comment]
@bsclifton I'm sorry. Didn't see your message. Yep, I mean this feature.

Before:
![image](https://cloud.githubusercontent.com/assets/17493891/18173114/0e1e6ef0-7071-11e6-8a67-b0607fcc6646.png)
After:
![image](https://cloud.githubusercontent.com/assets/17493891/18173153/35001b72-7071-11e6-8346-67cbe56eaf46.png)

**Comment by bsclifton** [comment]
@NyanTyrrell does the bookmarks toolbar go away once you've navigated to a page?

**Comment by NyanTyrrell** [comment]
@bsclifton Yep, they just disappear when you open a page. This feature saves space.

**Comment by bsclifton** [comment]
Awesome! Thanks for verifying, @NyanTyrrell :smile: This would be a great feature

**Comment by bradleyrichter** [comment]
I am concerned that we will have a lot of UI movement because our BM bar is above the tabs but maybe some CSS animation would soften it up. Certainly worth a test... Thanks @NyanTyrrell !
